app.rb

```diff
@@ -0,0 +1,152 @@
+
+class App
+    def initialize 
+        @store ={
+            person: []
+            books: []
+            rentals: []
+        }
+        @method={
+            '1': 'list_all_books',
+            '2': 'list_all_people',
+            '3': 'create_person',
+            '4': 'create_book',
+            '5': 'create_a_rental',
+            '6': 'list_rental_by_person_id',
+            '7': 'exit' 
+        }
+    end
+    def list_all_books
```

Comment:
The user experiences an error while clicking on option `1`. Kindly see the error below.
![error](https://user-images.githubusercontent.com/35504149/170672845-18b16cf1-147d-4aad-8cdd-a5de43452266.png)

Reply:
okay, you right i did fix